vnext/local-cli/src/runWindows/utils/deploy.ts

```diff
@@ -54,7 +59,16 @@ function getAppPackage(options: RunWindowsOptions): string {
   const appPackageGlob = `${
     options.root
   }/windows/{*/AppPackages,AppPackages/*}/${packageFolder}`;
-  let appPackage = glob.sync(appPackageGlob)[0];
+  const globs = glob.sync(appPackageGlob);
```

Comment:
Nit: globs describe the pattern, not the result. Consider something like "appPackageFiles" instead. 

Same below